ui/components/multichain/import-tokens-popover/import-tokens-popover.stories.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import configureStore from '../../../store/store';
+import testData from '../../../../.storybook/test-data';
+import { CHAIN_IDS } from '../../../../shared/constants/network';
+import { ImportTokensPopover } from './import-tokens-popover';
+
+const store = configureStore({
+  ...testData,
+  metamask: {
+    ...testData.metamask,
+    useTokenDetection: true,
+    providerConfig: {
+      chainId: CHAIN_IDS.MAINNET,
+    },
+  },
+});
+
+export default {
+  title: 'Components/Multichain/ImportTokensPopover',
+  component: ImportTokensPopover,
+  argTypes: {
+    onClose: {
+      action: 'onClose',
+    },
+  },
+  args: {
+    closeMenu: () => console.log('Closing popover!'),
+  },
```

Comment:
I think this can be removed? onClose is covered in the actions tab


<img width="1144" alt="Screenshot 2023-06-26 at 1 36 11 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/e61cbb86-4753-4e7b-aa0d-f8f5fab6016c">